src/pages/home/sidebar/AllSettingsScreen.tsx

```diff
@@ -49,10 +50,10 @@ function AllSettingsScreen({policies}: AllSettingsScreenProps) {
                 focused: !isSmallScreenWidth,
                 brickRoadIndicator: hasGlobalWorkspaceSettingsRBR(policies) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
             },
-            ...(shouldShowSubscriptionsMenu
+            ...(privateSubscription
```

Comment:
Not considering an active subscription plan based on control/collect workspace for display of `subscription` menu resulted in #45720